packages/modules/itwin-viewer-react/src/components/iModel/IModelLoader.tsx

```diff
@@ -113,8 +113,10 @@ const Loader = React.memo(
             uiConfig
           );
 
-          await SelectionScopeClient.initializeSelectionScope();
-          SelectionScopeClient.setupSelectionScopeHandler();
+          if (!snapshotPath) {
```

Comment:
any reason SelectionScope doesnt work with snapshots?

Reply:
combination of us using settings in the client and issues in the sync event dispatcher. Going to remove the settings requests across the viewer in a PR this week and will revisit if we can re-enable at that point